doc/source/getting_started/write_doc/index.rst

```diff
@@ -0,0 +1,138 @@
+
+Here is an overview of how our documentation is organized to help you know where you should include your contributions.
+Each section has their own guidelines that must be followed when creating new content.
+
+.. grid:: 1 1 2 2
+    :gutter: 2
+    :padding: 2
+    :margin: 2
+
+    .. grid-item-card:: **TUTORIALS**
+       :link: ref_guidelines_tutorials
+       :link-type: ref
+       :class-title: sd-text-center sd-bg-light
+       :class-header: sd-text-center
+
+       Learning oriented
+       ^^^^^^^^^^^^^^^^^
+
+       **Function:**  Teach how to get started and use PYDPF-core step by step
+
+       They are designed to teach how to perform a task and understand the underlying concepts,
+       providing detailed explanations at each stage.
+
+       +++
+       .. rubric:: Guidelines
+
+       Here you find guidelines and templates to write new tutorials.
+
+    .. grid-item-card:: **EXAMPLES**
+       :link: ref
+       :link-type: ref
+       :class-title: sd-text-center sd-bg-light
+       :class-header: sd-text-center
+
+       Use-cases oriented
+       ^^^^^^^^^^^^^^^^^^
+
+       **Function:**  Show how to solve specifics key problems
+
+       They showcase specific key problems and use-cases. They are more advanced than
+       tutorials and assume some knowledge on PyDPF-Core.
+
+       +++
+       .. rubric:: Guidelines
+
+       Here you find guidelines and templates to write new examples.
+
+    .. grid-item-card:: **CONCEPTS**
+       :link: ref
+       :link-type: ref
+       :class-title: sd-text-center sd-bg-light
+       :class-header: sd-text-center
+
+       Understanding oriented
+       ^^^^^^^^^^^^^^^^^^^^^^
+
+       **Function:**  Provide useful background explanation on PyDPF-Core
+
+       They discuss and explain key topics and concepts providing enabling the reader to understand our
+       tool.
+
+       +++
+       .. rubric:: Guidelines
+
+       Here you find guidelines and templates to write more concepts.
+
+
+    .. grid-item-card:: **API REFERENCE**
+       :link: ref
+       :link-type: ref
+       :class-title: sd-text-center sd-bg-light
+       :class-header: sd-text-center
+
+       Informing oriented
+       ^^^^^^^^^^^^^^^^^^
+
+       **Function:** Describe PyDPF-Core APIs
+
+       They contain technical reference on how PyDPF-Core works and how to use it but assume that you have
+       a basic understanding of key concepts.
+
+       +++
+       .. rubric:: Guidelines
+
+       Here you find guidelines and templates to improve the API reference.
+
+View the documentation
+----------------------
+
+Documentation for the latest stable release of PyDPF-Core is hosted at
+`PyDPF-Core Documentation <https://dpf.docs.pyansys.com/>`_.
+
+In the upper right corner of the documentation's title bar, there is an option
+for switching from viewing the documentation for the latest stable release
+to viewing the documentation for the development version or previously
+released versions.
```

Comment:
@luisaFelixSalles no, here we need to explain how to generate the documentation locally.
We will most likely have a dedicated page in the `Getting Started`, and the process for easily generating the documentation is being worked on right now, so it is not ready.
We could explain how to build it locally as of today, but that is not the point of this PR.